internal/project/project.go

```diff
@@ -410,6 +410,11 @@ func getCodeBlocksFromFile(path string) (document.CodeBlocks, error) {
 func getCodeBlocks(data []byte) (document.CodeBlocks, error) {
 	identityResolver := identity.NewResolver(identity.DefaultLifecycleIdentity)
 	d := document.New(data, identityResolver)
+
+	if f, _ := d.Frontmatter(); f != nil && f.Runme.Session.ID != "" {
+		return document.CodeBlocks{}, nil
```

Comment:
Hint: This is more idiomatic and does not allocate any memory (negligible in this case):

```suggestion
		return nil, nil
```